### PR TITLE
fixes https://github.com/Frijol/PIR/issues/5

### PR DESCRIPTION
**Pull request by Frijol**
Adds Tessel 2 connection instructions. @ahetawal does this help? Any suggestions?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +62,7 @@ the field.
 *********************************************/
 
 var tessel = require('tessel');
-var pir = require('../').use(tessel.port['GPIO'].pin['G3']);
+var pir = require('../').use(tessel.port['A'].pin[1]);
```
Comment (ahetawal) [inline README.md]: Will it be pin1 or pin3 here ?

File: README.md [hunk 2]
```diff
@@ -13,30 +13,47 @@ Adafruit has a lovely writeup on how PIRs work; I encourage you to read the deta
...
+I've left the PIR sensor in the retriggering position ([H for the Adafruit one](https://learn.adafruit.com/pir-passive-infrared-proximity-motion-sensor/testing-a-pir)), so the signal pin will be pulled high as long as the sensor detects movement.
+
+### Tessel 1 connection example
+
 ![](https://lh3.googleusercontent.com/-Zcmg7NPefAM/U-LafNQe2lI/AAAAAAAAJno/jlCWeBbDWxU/w882-h496-no/20140801_113753.jpg)
 
-I've left the PIR sensor in the retriggering position ([H for the Adafruit one](https://learn.adafruit.com/pir-passive-infrared-proximity-motion-sensor/testing-a-pir)), so the signal pin will be pulled high as long as the sensor detects movement.
+In this image, the black wire is ground; the red wire is 5V; and the yellow wire is the signal pin.
+
+This is connected to `tessel.port['GPIO'].pin('G3')`.
```
Comment (ahetawal) [inline README.md]: May be this goes on same line as above where you mention signal pin.

File: examples/pir.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ the field.
 *********************************************/
 
 var tessel = require('tessel');
-var pir = require('../').use(tessel.port['GPIO'].pin['G3']);
+var pir = require('../').use(tessel.port['A'].pin[1]);
```
Comment (ahetawal) [inline examples/pir.js]: Same pin1 or pin3, either update the picture and the example text in readme.

**Comment by ahetawal** [comment]
Added my comments, but overall looks very helpful and much clearer to follow. Will give it a shot once all my hardware arrives.

File: README.md (hunk 1 above)
Comment (Frijol) [inline README.md]: it doesn't matter; any [digital pin](https://tessel.io/docs/hardwareAPI#pin-mapping) will work. Is there somewhere this should be made more clear?

File: examples/pir.js (hunk 3 above)
Comment (Frijol) [inline examples/pir.js]: oh, I see - to match the image? Can do.

**Comment by Frijol** [comment]
cool! I'll leave this open until the package actually works properly on T2. Let me know how it goes with the hardware!

**Comment by Frijol** [comment]
whoops, per https://github.com/tessel/t2-firmware/issues/71 I should probably use pin 2 in the examples

**Pull request closed without merging**